tikv/mvcc_test.go

```diff
@@ -1387,16 +1387,16 @@ func (s *testMvccSuite) TestResolveCommit(c *C) {
 	// The error path
 	kvTxn := store.MvccStore.db.NewTransaction(true)
 	e := &badger.Entry{
-		Key: y.KeyWithTs(sk, 2),
+		Key: y.KeyWithTs(sk, 3),
```

Comment:
This key was written with version 2, increase the version to 3 because badger does not support update key with the same version.